.github/workflows/build.yml

```diff
@@ -67,15 +73,38 @@ jobs:
           echo "::set-env name=GOPATH::$(go env GOPATH)"
           echo "::add-path::$(go env GOPATH)/bin"
       - name: Verify
-        # disabled for windows due to format verification failing
-        if: matrix.os != 'windows'
         run: make verify
+      - name: Test-WCOW
+        shell: powershell
+        if: matrix.config == 'windows-wcow'
+        env:
+          TEST_COVERAGE: 1
+          # https://github.com/buildpacks/lifecycle/actions/runs/159882275
+          # https://github.com/buildpacks/lifecycle/suites/883071116/artifacts/10407089a
+          # LIFECYCLE_URL temporarily retrieved from the following until the windows lifecycle binary is released
+          LIFECYCLE_URL: https://windows-bosh-temp.s3.amazonaws.com/lifecycle-windows-x86-64.zip
```

Comment:
The artifacts required a login, so we opted for a simpler solution temporarily. Realistically, lifecycle is to be released today (or early next week) with Windows support, so we'll likely be using that.

Additionally/alternately, we can leave this as is for this PR and remove it as part of #738, which might make more sense.

Reply:
As yet another alternative, we could prioritize getting #738 merged before this PR, since that one really has to do with `create-builder` functionality, which has been in `main` for a few weeks now.

Reply:
We're going to go with the latest approach and get #738 in first, then we'll rebase this